travis.sh

```diff
@@ -96,7 +89,6 @@ function run_docker() {
         -v ${CCACHE_DIR:-$HOME/.ccache}:/root/.ccache \
         -t \
         -w /root/$REPOSITORY_NAME \
-        "${run_opts[@]}" \
```

Comment:
I think it is important to pass SSH_AUTH_SOCK into docker, because otherwise the credentials will be only available in Travis.